example/service/dynamodb/unitTest/unitTest.go

```diff
@@ -0,0 +1,42 @@
+// Package unitTest demonstrates how to unit test, without needing to pass a
+// connector to every function, code that uses DynamoDB.
+
+// +build example
```

Comment:
The `// +build example` should be the first line of a file followed by a empty line. After that you can start the package docs.

```go
// +build example

// Package unitTest demonstrates how to unit test, without needing to pass a
// connector to every function, code that uses DynamoDB.
```